File: app/src/main/java/com/tonitealive/app/ui/presenters/signin/SignInPresenter.java
package com.tonitealive.app.ui.presenters.signin;

import com.tonitealive.app.internal.di.annotations.PerActivity;

@PerActivity
public interface SignInPresenter {
    void onResume();
    void onSignInButtonClicked();
    void onSignUpButtonClicked();
    void onForgotPasswordButtonClicked();
}

File: app/src/main/kotlin/com/tonitealive/app/ui/presenters/signin/SignInPresenter.kt
package com.tonitealive.app.ui.presenters.signin

import com.tonitealive.app.internal.di.annotations.PerActivity

@PerActivity
interface SignInPresenter {
    fun onSignInButtonClicked()
    fun onSignUpButtonClicked()
    fun onForgotPasswordButtonClicked()
}